### PR TITLE
Added Micro-Enterprise

### PR DESCRIPTION
**Pull request by Rahul-Ravindran-Official**

### CODE REVIEW

File: .vuepress/config.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ module.exports = {
                     { text: 'Projects', link: '/projects/' },
                     {
                         text: 'Communities', items: [
+                            { text: 'Micro-Enterprise', link: '/micro-enterprise/' },
```
Comment (ArsalaBangash) [inline .vuepress/config.js]: Please put your link below the other communities.

File: micro-enterprise/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## About
+Micro-Enterprise is an initiative to Orchistrate a complete professional lifecycle for UofT students.
+
+Micro-Enterprise aims to provide consistant and guarenteed internship opportunities for all MCS students but this summer, we aim to pilot this with a small subset of students and development processes.
+
+## Benefits
++ Get an internship by just working for a minimum of 5 hours a week.
++ Get guarenteed internship letters upon successfully completing projects.
++ Learn the art of enterprise-grade software delivery.
++ We're aligning with more and more companies and MNCs everyday, to get this program recognised. With your consent get a chance to be hired by them upon successfully completing projects.
```
Comment (ArsalaBangash) [inline micro-enterprise/README.md]: How about: 

"You get the chance to be hired by our ever-updating list of companies and MNCs"

You already mention before that you are guaranteed internship letters upon successfully completing projects.

File: micro-enterprise/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
++ Upon successfully completing a project, you would always get a guarenteed internship every semester giving you the ability to rack upto 4 internships every semester and therefore upto 12 internships per year. Thereby catapulting your career for success.
```
Comment (ArsalaBangash) [inline micro-enterprise/README.md]: How about: 

"Catapult your career for success by taking part in potentially 4 internships per semester, or 12 internships per year."

File: micro-enterprise/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Apply Now
+Application will open between May 1 - May 3. If you are interested in being a part of Micro-Enterprise please add yourself to the mailing list where we will notify you when the applications unfold and other cool updates to propel your career-readiness.
```
Comment (ArsalaBangash) [inline micro-enterprise/README.md]: Applications*

Comment (ArsalaBangash) [inline micro-enterprise/README.md]: If you are interested in being a part of Micro-Enterprise please **join our** mailing list **to get notified** when applications open and when we have cool updates for you to propel your career-readiness.

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]

File: .vuepress/config.js (hunk 1 above)
Comment (Rahul-Ravindran-Official) [reply .vuepress/config.js]: Position has been changed as advised in the revised commit. Please review the latest commit.

File: micro-enterprise/README.md (hunk 2 above)
Comment (Rahul-Ravindran-Official) [reply micro-enterprise/README.md]: Change has been incorporated with in the revised commit. Please review the latest commit.

File: micro-enterprise/README.md (hunk 3 above)
Comment (Rahul-Ravindran-Official) [reply micro-enterprise/README.md]: Change has been incorporated with in the revised commit. Please review the latest commit.

File: micro-enterprise/README.md (hunk 4 above)
Comment (Rahul-Ravindran-Official) [reply micro-enterprise/README.md]: Change has been incorporated with in the revised commit. Please review the latest commit.

Comment (Rahul-Ravindran-Official) [reply micro-enterprise/README.md]: Change has been incorporated with in the revised commit. Please review the latest commit.

**Comment by Rahul-Ravindran-Official** [comment]
Hi Arsala,
All changes has been incorporated. I request you to review the changes.

**Review by ArsalaBangash — APPROVED** [review]
Excellent work!

**Pull request merged**